### ISSUE TITLE
After successful install, `make test` fails

### ISSUE THREAD

**Issue by idiazroncero** [Metadrop/drupal-boilerplate#61]
Output:

```sh
docker-compose exec php phpunit
[25-Feb-2022 13:36:41 UTC] PHP Fatal error:  Trait 'Symfony\Bridge\PhpUnit\Legacy\PolyfillAssertTrait' not found in /var/www/html/web/sites/simpletest/Assert.php on line 91

Fatal error: Trait 'Symfony\Bridge\PhpUnit\Legacy\PolyfillAssertTrait' not found in /var/www/html/web/sites/simpletest/Assert.php on line 91
make: *** [Makefile:24: test] Error 255
```

`/var/www/html/web/sites/simpletest/Assert.php` exists

Following [this issue](https://www.drupal.org/project/drupal/issues/3220043) and executing `composer require symfony/phpunit-bridge` makes it work but results on a new error

```
Test file "/var/www/html/./tests/TestSuites/UnitTestSuite.php" not found
make: *** [Makefile:24: test] Error 2
```

(a new file, `web/sites/simpletest/TestCase.php`, is generated when executing this command)

Running on a Kubuntu impish 21.10 kernel version 5.13.0-30-generic, python version is 2.7.18.
drupal-boilperlate version was 2.0.0:

```
Installing metadrop/drupal-boilerplate (v2.0.0)
  - Installing metadrop/drupal-boilerplate (v2.0.0): Extracting archive
```

**Comment by rsanzante** [comment]
Confirmed.

**Comment by rsanzante** [comment]
Fixed in  v2.0.0-alpha3
